crawler/features_crawler.py

```diff
@@ -82,6 +85,12 @@ def __init__(
         self.crawl_mode = crawl_mode
         self.container = container
         self.vm = vm
+        self.vm_context = vm_context
+
+        if crawl_mode == Modes.OUTVM:
+            global psvmi 
+            psvmi = __import__('psvmi')
```

Comment:
What about something like this. You import psvmi at the top of this file like this:
```
try:
    import psvmi
except ImportError:
    psvmi = None
```

and then here, you check `psvmi` not None every time you use it. This way you won't have to import it every time.